include/exadg/postprocessor/solution_field.h

```diff
@@ -109,28 +128,11 @@ class SolutionField : public dealii::Subscriptor
   dealii::DoFHandler<dim> const * dof_handler;
 
 private:
-  bool               is_available;
-  VectorType         vector;
-  VectorType const * solution;
+  mutable bool                        is_available;
+  mutable std::shared_ptr<VectorType> solution_vector;
+  VectorType const *                  src_vector;
```

Comment:
I think src_vector is not used in your current implementations anymore.

Reply:
agree